.github/workflows/CI.yml

```diff
@@ -20,3 +20,14 @@ jobs:
         run: |
           cmake .
           cmake --build . --target check
+      - name: Test Coverage
```

Comment:
Maybe we should put this in a separate workflow file. Then, we could probably simplify the matrix in that file.